main.go

```diff
@@ -695,6 +709,32 @@ func main() {
 
 		go runCommands(startCommand, server, false, false)
 	}
+
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM, syscall.SIGINT)
+	go func() {
+		<-c
+		cmps.mu.Lock()
+		defer cmps.mu.Unlock()
+		ControllerInfo.log("Inside signal handler for controller")
```

Comment:
I think all of these Info messages in the signal handler should be Debug level.

Maybe add just one Info message like "Interrupt signal received, shutting down processes."

Reply:
changes made